includes/fbproduct.php

```diff
@@ -534,6 +534,17 @@ public function prepare_product( $retailer_id = null, $prepare_for_product_feed
 				'visibility'            => Products::is_product_visible( $this->woo_product ) ? \WC_Facebookcommerce_Integration::FB_SHOP_PRODUCT_VISIBLE : \WC_Facebookcommerce_Integration::FB_SHOP_PRODUCT_HIDDEN,
 			);
 
+			// add the Commerce values
+			if ( Products::is_product_ready_for_commerce( $this->woo_product ) ) {
```

Comment:
From the discovery doc, I thought we were going to send these values regardless of Commerce being enabled (except for the Google product category ID). Have we decided to change that?

Reply:
While adding this I realized that it may not be a good idea to do that, after going over the frontend implementation. If the product is not ready or disabled for Commerce, the settings field for gender can't be changed. Folks could end up with products that send along values that they no longer want to use for gender, and inventory isn't relevant outside of the Commerce context.

Reply:
Sounds good, we should probably update the doc to reflect that decision. 😬

Reply:
Updated!